### ISSUE TITLE
M5: training runbook + eval comparison of the trained model

### ISSUE THREAD

**Issue by thodel** [thodel/serving-atr-inference#37]
Part of the training subsystem — plan: [`docs/TRAINING_PLAN.md`](https://github.com/thodel/serving-atr-inference/blob/main/docs/TRAINING_PLAN.md). Depends on #36.

Close the loop: document the runbook, and make a trained model measurable against the models already served.

### Docs

`docs/TRAINING.md` — the operator runbook, in the style of `docs/DEPLOY.md`:

- building `.venvs/kraken-train` and installing `atr-train.service`
- submitting a job (curl + body), reading status, tailing a stage log, cancelling
- the job directory layout under `~/atr-cache/training/<job_id>/` and what to keep vs. delete
- how a trained model gets registered and promoted, and how to un-register one
- **the disk story**: the GT dataset is ~6.6 TB across 694 per-project parquet dirs and the box has ~356 GB free, so subsets are always selected by `data_files` glob; how to check what a job will materialize before starting it
- troubleshooting: OOM at batch 256 → `-B 64 --accumulate-grad-batches 4`; a `1cycle` run cut short by early stopping; a job left `training` with a dead PID

`docs/TRAINING_PLAN.md` gets the measured numbers folded back in (peak VRAM, wall time, CER) so the plan stops being a forecast.

### Evaluation

Hook the trained model into `eval/run_eval.py` so it can be scored on the same held-out pages as the served kraken models, and record the comparison. Note that `ketos test` CER and the eval harness CER are **not** the same measurement — `ketos test` scores line crops from ground-truth segmentation, the harness scores a full page through our own segmentation. Report both, labelled, rather than picking whichever looks better.

### Acceptance

- [ ] `docs/TRAINING.md` walks a reader from an empty box to a trained, served, scored model
- [ ] Eval report comparing the Thun model against the served kraken models on held-out Thun pages, with both CER numbers labelled by how they were obtained
- [ ] README links to the training docs
- [ ] Follow-ups filed as separate issues: TrOCR fine-tuning backend, VLM LoRA fine-tuning, `ketos publish` / HF upload, `agentic_historian` client wiring

**Comment by thodel** [comment]
Baseline comparison done for the first VLM run — `evaluate_qlora.py` gained a `--no-adapter` mode (6e3d6c6) so the un-adapted base can be scored on the same samples.

Same 25 val samples, same prompt, same budgets, job `20260808T080206Z-qwen3vl-thun-smoke`:

| | CER | WER |
|---|---:|---:|
| Qwen3-VL-8B-Instruct, un-adapted | 1.837 | 2.386 |
| + 38 steps of QLoRA | **0.466** | **0.816** |

**But most of that is not what it looks like.** A CER above 1 means the base emitted far more characters than the reference, and the examples show why — it does not stop at the line:

```
ref : Sigriswil und von Stefisburg gegen den unnsern von hann
base: Gestandene Erkennung der Tatsachen, daß sie von Digriftel und von
      Stüffseng gegen den Vormund von Tömy dabey gebraucht haben und
      darum wohl als geliehene Kost
```

One crop, one line of ground truth, several lines of output drifting into paraphrase. It reads a fair amount on the way — for `wir haben verstanden die ordnung der versuͦchen so die von` its second output line was `Von haben voranstanden diefenbedenung der Kurfusten/So die van`.

So 38 steps mainly taught **output discipline** — emit one transcription, stop — which dominates an edit-distance metric when the baseline over-generates 2–3×. How much better it actually *reads* is not answered by this comparison.

### What that means for this issue

A one-number before/after is misleading for VLM backends specifically, in a way it is not for kraken (which cannot over-generate — it decodes a fixed-width line). If `eval/run_eval.py` is going to compare a trained model against the served ones, it needs at least one of:

- **length-controlled scoring** alongside raw CER, so format compliance and recognition are separable;
- or a **constrained baseline** (stop at first newline / `max_new_tokens` near the reference length), so both sides are answering the same question.

Recorded in `docs/VLM_TRAINING.md` (5098635) with the caveat attached, so the 74 % is not quoted bare.
